From 198bf30ee410fb0622c6fa53a11c2a89cc71023d Mon Sep 17 00:00:00 2001
From: Yehuda Yitschak <yehuday@marvell.com>
Date: Mon, 28 Jul 2014 15:51:00 +0300
Subject: [PATCH 0045/1240] mvebu: config: Modified general configuration for
 mvebu

	- Reduced malloc size to 1MB
	- Removed init ramdisk from bootcmd
	- Added option to enable boot to EL1

Change-Id: I32705f105fc1d1f14fe0a74ef5dd1801775c00cd
Signed-off-by: Yehuda Yitschak <yehuday@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/9685
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 include/configs/mvebu-common.h | 9 ++++++---
 1 file changed, 6 insertions(+), 3 deletions(-)

diff --git a/include/configs/mvebu-common.h b/include/configs/mvebu-common.h
index 0fde7c7..e900113 100644
--- a/include/configs/mvebu-common.h
+++ b/include/configs/mvebu-common.h
@@ -29,7 +29,7 @@
 /* Temp */
 #define CONFIG_ADEC
 
-/* Udefull Debug */
+/* Usefull Debug */
 #define DEBUG
 #define CONFIG_BOOTSTAGE
 #define CONFIG_BOOTSTAGE_REPORT
@@ -43,7 +43,7 @@
 #define CONFIG_SYS_SDRAM_BASE           0x00000000
 #define CONFIG_SYS_RESET_ADDRESS        0xffff0000
 #define CONFIG_SYS_MALLOC_BASE          (CONFIG_SYS_TEXT_BASE + (3 << 20))  /* TEXT_BASE + 3M */
-#define CONFIG_SYS_MALLOC_LEN           (5 << 20)    /* Reserve 5MB for malloc*/
+#define CONFIG_SYS_MALLOC_LEN           (1 << 20)    /* Reserve 1MB for malloc*/
 #define CONFIG_NR_DRAM_BANKS		(4)
 /* maybe need to set back to 0x7fff0 */
 #define CONFIG_SYS_INIT_SP_ADDR         (CONFIG_SYS_TEXT_BASE + 0xFF0000)   /* End of 16M scrubbed by training in bootrom */
@@ -119,7 +119,7 @@
 					"fdt_high=0xa0000000\0"
 
 #define CONFIG_BOOTARGS			"console=ttyS0,115200 earlyprintk root=/dev/ram0"
-#define CONFIG_BOOTCOMMAND		"bootm $kernel_addr $initrd_addr:$initrd_size $fdt_addr"
+#define CONFIG_BOOTCOMMAND		"bootm $kernel_addr - $fdt_addr"
 
 /* Marvell specific env*/
 #define MV_BOOTARGS_END			":10.4.50.254:255.255.255.0:Marvell:eth0:none"
@@ -156,6 +156,9 @@
 /* SMP Spin Table Definitions */
 #define CPU_RELEASE_ADDR               (CONFIG_SYS_SDRAM_BASE + 0x7fff0)
 
+/* Exception level */
+/* define CONFIG_ARMV8_SWITCH_TO_EL1 */
+
 /*********  IO Definitions **********/
 
 /* UART */
-- 
1.9.1

